src/librustdoc/lib.rs

```diff
@@ -366,7 +366,7 @@ fn opts() -> Vec<RustcOptGroup> {
             )
         }),
         unstable("Z", |o| {
-            o.optmulti("Z", "", "internal and debugging options (only on nightly build)", "FLAG")
+            o.optmulti("Z", "", "unstable / perma-unstable options (only on nightly build)", "FLAG")
```

Comment:
this should probably have a rustdoc reviewer - @camelid does this new description look good?